docs/tutorials/avro.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<p><code>json.dumps</code>  is being used instead of <code>dumps</code>  so this import can be removed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/io/pull/1250/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
thanks, updated.